pkg/kubelet/kubelet.go

```diff
@@ -3014,7 +3014,7 @@ func (kl *Kubelet) RunInContainer(podFullName string, podUID types.UID, containe
 
 	var buffer bytes.Buffer
 	output := ioutils.WriteCloserWrapper(&buffer)
-	err = kl.runner.ExecInContainer(container.ID, cmd, nil, output, output, false, nil)
+	err = kl.runner.ExecInContainer(container.ID, cmd, nil, output, output, false, nil, 0)
```

Comment:
Do you think we should go add another argument to `RunInContainer`?
If that method is "the old way", I would just keep it as is, no timeout support.